docs/users.py

```diff
@@ -28,3 +28,10 @@
 find_user_by_identifier_summ = """
 Find user by identifier
 """
+
+login_user_desc = """
+Validates user against database using 
+"""
+login_user_summ = """
+Validates user
+"""
```

Comment:
Since the login process _is_ validation, would this suffice?

login_user_desc = """
Validates and logs in a user against database
"""
login_user_summ = """
Validates and logs in a user
"""

Reply:
Since these docs are going to the frontend team anyway we might as well speak their language; at least for the `summ`, "User login" would be more than good.